agents2/mock_data_generator.py

```
@@ -0,0 +1,252 @@
+"""
+Realistic Mock Data Generator for AI Agents
+==========================================
+
+Generates realistic mock responses for development without API costs.
+"""
+
+import random
+import time
```

Ruff (0.12.2)

`time` imported but unused

Remove unused import: `time`

(F401)